### ISSUE TITLE
ttyplay displays some gargabe sometimes

### ISSUE THREAD

**Issue by soichih** [jedi4ever/ttyrec.js#1]
Hi. I am playing with ttyrec/play and see what it can do. So far it's quite fun!

I've noticed however, that it sometimes display some artifacts on the terminal while I am re-playing what has been recorded.

Like this..

> ^[[>0;136;0c
> 
> 2014-01-04 15:21:47 UTC [hayashis@soichi6:~]$ 0;136;0c

This also happens when ttyplay plays back my vim editor. I think it's trying to reset the terminal color or width or something and my tty doesn't understand it. I am using RHEL6.4 on bash terminal. I am using putty on Windows 8. This could be just a putty issue.. but I am not sure.

**Comment by jedi4ever** [comment]
thanks @soichih for reporting this. ttyrec/ttyplay by itself does not do anything with the escape codes.

To see what's happening I would like:
- the ttyrecord of the recording (so I can check if it happens with me)
- do I get it correct that you use putty to login into a RHEL6.4 machine with a bash terminal?
- I'm wondering if the TERM setting of the remote machine and the local machine differ?

P.S. Also update the package as I've made some commandline args options to the ttyrec/ttyplay command;

**Comment by soichih** [comment]
Jedi4ever,

I've posted my recording here > http://soichi6.grid.iu.edu/public/ttyrecord

When I play it back, the vim screen is displayed corrupted with the escape characters I mentioned earlier.

Yes, I am using putty on Windows 8 to ssh to my RHEL6.5 machine which uses bash shell by default. 

I see following set for TERM

2014-01-05 15:34:46 UTC [hayashis@soichi6:~]$ set | grep TERM
TERM=xterm

**Comment by jedi4ever** [comment]
I can replay this correctly.

What terminal did you set in Putty? I'm guessing it might not be xterm but vt100 or the likes.

**Comment by soichih** [comment]
"Function keys and keypad" was set to "ESC[n~" whatever that measn.. I found an option "Xterm R6" so, I tried that, but I didn't notice any difference. I am guessing this option just applies to how function key works. I don't ever use function key while I am on the terminal.

I think more promising settings to tinker with is the Windows / Translation page on Putty, but I couldn't find anything about xterm or such. 

Just by Googling "^[[>0;136;0c", I see a few other people seeing similar issues. I don't think this issue is related to ttyrec.js, but more of the terminal configuration issue.
